sys/net/network_layer/ng_ipv6/netif/ng_ipv6_netif.c

```diff
@@ -69,22 +64,27 @@ static int _add_addr_to_entry(ng_ipv6_netif_t *entry, const ng_ipv6_addr_t *addr
                     ll_addr.u64[1] = addr->u64[1];
                     ng_ipv6_addr_set_link_local_prefix(&ll_addr);
 
-                    entry->addrs[i].flags = NG_IPV6_NETIF_ADDR_FLAGS_UNICAST;
-
-                    return _add_addr_to_entry(entry, &ll_addr, 64, false);
+                    _add_addr_to_entry(entry, &ll_addr, 64,
+                                       flags | NG_IPV6_NETIF_ADDR_FLAGS_NDP_ON_LINK);
```

Comment:
Just a reminder: https://github.com/RIOT-OS/RIOT/pull/2909#discussion_r30003992 triggered this change.